packages/orchestration/src/__mocks__/joiner/mock_data.ts

```diff
@@ -75,6 +78,12 @@ export const serviceConfigs: JoinerServiceConfig[] = [
   },
   {
     serviceName: "order",
+    fieldAlias: {
```

Comment:
question: Can we also alias deeply nested properties?

For example, consider the following query: 

```
query {
  lineItems {
    variant {
      product {
        id
        title
        shipping_profiles {
          profile
        }
      }
    }
  }
}
```

Can I alias the shipping_profile to be queryable as just profile "inside" the product?

Like so:

```
query {
  lineItems {
    variant {
      product {
        id
        title
        profile
      }
    }
  }
}
```

Reply:
At the moment the nested fields are mandatory to be selected. The query would become this: 

```graphql
query {
  lineItems {
    variant {
      product {
        id
        title
        profile {
           id,
           name
        }
      }
    }
  }
}
```

the config would be very similar to what is at the description.
To create aliases at the field level it would require to know the entire schema/fields, or change the way the config is done. If needed we are closer to have it working

Reply:
But to answer, yes, it can be an alias to a deep nested field. No matter how many levels, it will build all the services dependency using their pk/fk keys.

Reply:
That's great. Exactly what I was after